LICENSE

```diff
@@ -176,7 +175,18 @@
 
    END OF TERMS AND CONDITIONS
 
-   Copyright 2015 The Linux Foundation.
```

Comment:
We don't need to change the current 'LICENSE' to a template 'LICENSE' o we will lose our `copyright owner`